Moonshot/player/LedgeWallDetector.gd

```diff
@@ -15,7 +15,7 @@ func _ready():
 	assert(ray_middle.cast_to.x >= 0)
 	assert(ray_bottom.cast_to.x >= 0)
 
-func _physics_process(delta):
+func _process(delta) -> void:
```

Comment:
Think this should be `_physics_process` 🤔

Reply:
@kepler471 ?

Reply:
This was a fix, will have to keep an eye on it though :)